doc/bookkeeper-options.md

```diff
@@ -67,7 +67,7 @@ Default gcOpts:
 "-XX:-ResizePLAB",
 ```
 
-Default gcLoggingOpts:
+Due to disruptive changes in GC Logging from Java 9, the BookKeeper operator code does not add any default gcLoggingOpts. The appropriate gcLoggingOpts can be provided based on the Java version used within the Bookkeeper version being deployed. If the Java version is 8 or lower, the following options can be provided:
```

Comment:
Is this true? The values.yaml file in this PR is actually setting some GC logging options, right? If you don't want to add any GC logging option, then you need to comment also the JDK11 ones in the values.yaml.

Reply:
The options present in the values.yaml can be easily overridden by the users which is why I have refrained from mentioned those options as the defaults added by the operator.